lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,81 @@
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
+- *test* - the number of the fuel efficiency test
+- *model* - the car used in the test (A-F)
+|
+- *speed* - the speed at which the car was tested
+- *mpg* - the miles-per-gallon the car achieved at that speed
+|===
+
+@n Evaluate `A15`, `A45` and `A75` in the Interactions Area. What *model* of car is used in all three rows? @fitb{}{@ifsoln{`A`}}
+
+@n At what three *speeds* is this model being tested in these rows? @fitb{}{@ifsoln{15, 45 and 75 miles per gallon}}
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="1a,1a", options="frame"]
+|===
+|
+@n In Pyret, make a scatter plot showing the relationship between `speed` and `mpg`, using `model` as your labels, then sketch the resulting scatter plot below. Be sure to label your axes!
+
+@ifsoln{@image{../images/fuel-efficiency-quadratic.png}}
+
+|
+Based on the scatter plot... +
+
+@n What do you *Notice?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What do you *Wonder?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n Which model is generally the most fuel-efficient? @fitb{}{@ifsoln{B}}
+
+@n **What speed is generally the most fuel-efficient?** @fitb{}{@ifsoln{45-55mpg}}
+
+|===
+
+@n Do you see a correlation? @fitb{5em}{@ifsoln{Yes}}. If so, describe its form (e.g. - linear, non-linear, or none) and strength (strong, weak, moderate, or none). If it's linear, is the correlation positive, negative, or zero? If it's not linear, describe its shape.
+
+@fitb{}{@ifsoln{There seems to be a moderate/strong, non-linear correlation}}
+
+@fitb{}{@ifsoln{It looks like a parabola that opens down, with a dimple at the top}}
+
+@center{**Stop here! We'll continue after some discussion.**}
+
+@n A linear model is defined on line 22, with its Contract on line 21. What is its name? @fitb{2em}{@ifsoln{f}} Domain? @fitb{6em}{@ifsoln{Number}} Range? @fitb{}{@ifsoln{Number}}
+
+@n Use the `fit-model` function (check your contracts page!) to fit the linear model `f` to the data. What is the @math{R^2} value? @fitb{}{@ifsoln{-11.794}}.
+
+
+@n Tweak the slope and y-intercept of `f` to improve the model, and record the resulting model below. What is the @math{R^2}? @fitb{}{}
+
+@n What does it tell us? @fitb{}{}
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{}{slope}x + @fitbruby{8em}{}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS varies) x) (?ANS varies))))}
+}
+
+@n Evaluate @show{(code '(lr-plot mpg-table "model" "speed" "mpg"))} to find the _best-possible_ linear model. What is the @math{R^2}? @fitb{5em}{@ifsoln{0.00426}}  Write the model below.
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{@ifsoln{@math{-0.01017}}}{slope}x + @fitbruby{8em}{@ifsoln{29.9459}}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS -0.01017) x) (?ANS 29.9459))))}
+}
+
+@fitb{}{}
+
+@n Could we get a better fit if we used a non-linear model? In your sketch (question 4, above), draw a curve that you think would fit better.
```

Comment:
Suggest rewording to:
_In your sketch (question 4, above), draw a curve that you think would fit. Do you think the linear or non-linear model you came up with fit better?._

Reply:
ooh, I like this a lot